### ISSUE TITLE
Kiln Not Working as intended

### ISSUE THREAD

**Issue by JaytheUnwise123** [svenhjol/CharmForge#39]
Charm version - 2.2.2
Minecraft version - 1.16.4

Items that can be smelted by the kiln do not smelt at twice the speed as normal. 
In fact, it seems it smelts at half the speed. It takes two of a one item smelting fuel to smelt one cobblestone for example.

**Comment by JaytheUnwise123** [comment]
I'm not sure if using twice the fuel is intended, sorry if it is. But it still doesn't smelt at twice the speed

**Comment by F43nd1r** [comment]
> In fact, it seems it smelts at half the speed.
It smelts much faster than a normal furnace in my testing.

> It takes two of a one item smelting fuel to smelt one cobblestone for example.
In my testing, both a furnace and a kiln take two sticks to smelt one cobblestone.

You might have other mods conflicting here.

**Comment by Syrikal** [comment]
I also have this issue. Items smelt at normal speed, but fuel lasts half as long as it should. Using Charm 2.2.2 for 1.16.4, Forge 35.1.28.

**Comment by littlepianist** [comment]
I also have the same issue as Syrikal does, the smelting speed of the kiln is the same as a normal furnace but the fuel requied is X2

**Comment by shondoa** [comment]
Same issue here. Kiln smelts at the same speed as a furnace but fuel depletes twice as fast.

**Comment by BrisingrAerowing** [comment]
https://github.com/svenhjol/CharmForge/blob/master/src/main/java/svenhjol/charm/tileentity/KilnTileEntity.java#L19

There's the problem. Dividing by 2 makes the fuel last half as long. I would presume you wanted to make it last twice as long.

**Comment by Syrikal** [comment]
I don't think so, actually. Fuel lasts half as long in blast furnaces and smokers as well- it's just that the items also smelt twice as fast, so you use the same amount of fuel but you smelt the items faster. The kiln's problem is that it's using fuel twice as fast but not smelting twice as fast.

**Comment by BrisingrAerowing** [comment]
Hm.

**Comment by Lawiel** [comment]
Up, having the same problem, it's smelting only 4 blocks for 1 coal

**Comment by JDLENL** [comment]
Having this issue still, charm-forge-1.16.5-2.3.2, on forge version 36.1.2; this is in a pack so there could be mod conflicts happening? But no other mod in the pack adds a kiln or anything similar.

**Comment by FynnWaelsing** [comment]
Do you have Polymorph installed? For some reason it causes this issue with Charm’s and Environmental’s kilns

**Comment by JDLENL** [comment]
Yes, but the pack does not have Environmental.

**Comment by FynnWaelsing** [comment]
That doesn’t matter. Do you have Polymorph? If so, try removing that.
